heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
+heron.class.uploader:         com.twitter.heron.uploader.docker.DockerUploader
+
+#base image
+heron.uploader.docker.base:                      ubuntu:latest
```

Comment:
The base here actually has to be a container with python installed in it. Aurora does some crazy stuff to get a aurora executor running inside the docker container so it can monitor its process and do health checking and things. The default ubuntu container does not have python installs so you'll need to set `python:2.7` or something similar as the base container.